src/Providers/ReviewsIo/ServiceActions.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace PodPoint\Reviews\Providers\ReviewsIo;
+
+use PodPoint\Reviews\ActionsInterface;
+use PodPoint\Reviews\ApiClientInterface;
+use PodPoint\Reviews\Exceptions\ValidationException;
+use PodPoint\Reviews\Providers\ReviewsIo\Request\Service\GetServiceReviews;
+use PodPoint\Reviews\Providers\ReviewsIo\Request\Service\EmailInviteRequest;
+use PodPoint\Reviews\Providers\ReviewsIo\Request\Service\FindReviewRequest;
+
+/**
+ * Class ServiceActions
+ * @package PodPoint\Reviews\Providers\ReviewsIo
+ */
+class ServiceActions implements ActionsInterface
+{
+    /**
+     * @var ApiClientInterface
+     */
+    protected $apiClient;
+
+    /**
+     * @var string
+     */
+    protected $store;
+
+    /**
+     * ServiceReview constructor.
+     *
+     * @param $apiClient
+     */
+    public function __construct(ApiClientInterface $apiClient)
+    {
+        $this->apiClient = $apiClient;
+    }
+
+    /**
+     * @param array $options
+     *
+     * @return array|mixed
+     * @throws ValidationException
+     */
+    public function invite(array $options)
+    {
+        $options['store'] = $this->store;
+
+        $request = new EmailInviteRequest($this->apiClient, $options);
+
+        return $request->send();
+    }
+
+    /**
+     * @param array $options
+     *
+     * @return array|mixed
+     * @throws ValidationException
+     */
+    public function getReviews(array $options)
+    {
+        $options['store'] = $this->store;
+
+        $request = new GetServiceReviews($this->apiClient, $options);
+
+        return $request->send();
+    }
+
+    /**
+     * Find reviews by id.
+     *
+     * @param string $reviewId
+     *
+     * @return array|mixed
+     * @throws ValidationException
+     */
+    public function findReview(string $reviewId)
+    {
+        $options = [
+            'reviewId' => $reviewId,
+            'store' => $this->store,
+        ];
+
+        $request = new FindReviewRequest($this->apiClient, $options);
+
+        return $request->send();
+    }
+
+    /**
+     * Sets store id and returns itself.
+     *
+     * @param string $store
+     * @return $this
+     */
+    public function setStore(string $store): self
+    {
+        $this->store = $store;
+
+        return $this;
+    }
+
+    /**
+     * Returns store id.
+     *
+     * @return string
+     */
+    public function getStore(): string
+    {
+        return $this->store;
+    }
+}
```

Comment:
I think that if we remove this, then have the store populated within:

```
PodPoint\Reviews\Providers\ReviewsIo\Request\Service\GetServiceReviews;
PodPoint\Reviews\Providers\ReviewsIo\Request\Service\EmailInviteRequest;
PodPoint\Reviews\Providers\ReviewsIo\Request\Service\FindReviewRequest;
```

then we can remove getStore, setStore, the corresponding tests for those, and we don't have to set the store in each function  within this class.

I think this will be cleaner, easier to maintain, and I think we can get away with it because the store-id is tied to the api-key, so I don't think there will be reason to change store when you are not changing api key <- I think I am understanding this correctly but as always please correct me if I'm missing something!!!